tests/e2e/support.cy.js

```diff
@@ -17,6 +17,12 @@ describe("Test the Cypress custom commands from 'support.js' file", () => {
     cy.visit('/administrator/index.php?option=com_banners&view=banners');
     cy.clickToolbarButton('new');
     cy.clickToolbarButton('cancel');
+    cy.visit('/administrator/index.php?option=com_content&task=article.add');
+    cy.get('#jform_title').clear().type('Test article versions');
+    cy.clickToolbarButton('Save');
+    cy.clickToolbarButton('Versions');
+    cy.get('.joomla-dialog-header').should('contain.text', 'Versions');
+    cy.task('queryDB', "DELETE FROM #__content WHERE title = 'Test article versions'");
```

Comment:
This should come into an afterEach block.

Reply:
but in that way we are executing the DELETE more than needed

Reply:
We should not mix that and never did in the tests.

Reply:
ok, moved